server/plugin.go

```diff
@@ -51,8 +51,10 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 }
 
 type analyticsData struct {
```

Comment:
Mind adding a GoDoc comment for this struct as it's am important struct, but not straight forward to see what it does.

Reply:
:heavy_check_mark: